optima/plotting.py

```diff
@@ -25,6 +25,40 @@
 
 
 
+def humanise_y_ticks(ax):
```

Comment:
i know we all have different conventions, but i would prefer to stick to a single convention within a project -- in optima so far we've been using US spellings and no underscores -- obviously for autumn and nutrition happy to comply with the existing convention for those